### PR TITLE
initial commit of signup page

### PR DESCRIPTION
**Pull request by Shadi-ya**

### CODE REVIEW

File: pages/login-page.html [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+        </svg>
+       <br><br>
+    <h1>Enter your email to join us or sign in.</h1>
+
+        <div class="details">
+          <form action="">
+            <div class="d-flex">
+              <p class="result" id="result-text">India</p>
+              <!-- <div class="select-country">
```
Comment (Amal911) [inline pages/login-page.html:40]: remove commented codes

File: pages/login-page.html [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+            <div class="d-flex justify-content-end">
+              <button class="btn btn-dark">Continue</button>
+            </div>
+          </form>
+        </div>
+      </div>
+    </section>
+    
+    <script>
+
```
Comment (Amal911) [inline pages/login-page.html:265]: move to js file

File: pages/login-page.html (hunk 1 above)
Comment (Shadi-ya) [reply pages/login-page.html]: Done

File: pages/login-page.html (hunk 2 above)
Comment (Shadi-ya) [reply pages/login-page.html]: Done

**Pull request merged**